ribs.docs

```diff
@@ -0,0 +1,26 @@
+# What is RIBS?
+A super small and easy to use collection of functions
+to help with some things pygame don't help with.
+
+Also note that not everything is documented, if it's not
+documented you probably don't have to care about it.
+
+
+# key_down(key)
+Takes a key, that's either a keycode or a character,
+and says if it's down or not.
```

Comment:
Might want to show both a keycode and a character being used. Arrow keys for example.